### ISSUE TITLE
SBR is running ARM in the cloud, how does this affect the rules?

### ISSUE THREAD

**Issue by sublayerio** [armada-alliance/armada-alliance#198]
@AstroWa3l SBR Pool is running ARM in the cloud, should we support flagging the nodes as "arm" so that the rules get checked?

> My stake pool is powered by Ampere Altra Q80-30 ARM low-power servers, provided by Oracle Cloud, Europe.

**Comment by zella** [comment]
How I undestand from "Our mission":

>  community of decentralized, **low-cost**, and **energy-efficient** stake pool operations

My cloud pool does not break this rule. Of course rule about ARM too.

But this question did open. So there should be a separate rule about "hosted at home". But this potential rule should have motivation, that be reflected in "Our Mission"

Whats about me - I choose cloud solution just because it was cheaper than buy rpi4. (Oracle cloud always free tier give 24gb ram and 4 core)

**Comment by AstroWa3l** [comment]
Yes, technically this pool satisfies the ARM requirements, but we will have to add new rules and fix the website wording IMO to make up for this situation. I have a few more rules I was thinking that would be able to show the difference between people who run in the cloud vs physical location they can access, low power consuming/renewable energy pools vs low power device in a massive cloud server farm, resiliency, and more.  We recommend clouds to be used as a last resort backup option, data storage, web hosting, etc. But using the cloud services to run pools even if you are using ARM still goes against the alliance moto of true decentralization.
